clang/include/clang/Driver/Options.td

```diff
@@ -2955,6 +2955,10 @@ defm clangir_direct_lowering : BoolFOption<"clangir-direct-lowering",
   FrontendOpts<"ClangIRDirectLowering">, DefaultTrue,
   PosFlag<SetTrue, [], [ClangOption, CC1Option], "Lower directly from ClangIR to LLVM">,
   NegFlag<SetFalse, [], [ClangOption, CC1Option],  "Lower through MLIR to LLVM">>;
+defm clangir_analysis_only_pipeline : BoolFOption<"clangir-analysis-only",
```

Comment:
The name `clangir_analysis_only` should be fine.

Reply:
Done